pkg/utils/select.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import "github.com/AlecAivazis/survey/v2"
+
+func SelectCOSInstance(instances []string) string {
```

Comment:
Let's make it more generic, so that can be used with any array by passing the message as param.

Reply:
Survey select accepts only string slice as an input 
https://github.com/AlecAivazis/survey/blob/a21fb70eb4d809c2e054eef36ad59167f5a67c2f/select.go#L24, Hence I am using data type string with the input param.

Reply:
as we discussed yesterday over a scrum, this is just a cosmetic change for making the function generic, logic stays same.

Reply:
Renamed function name to `func SelectItem(msg string, instances []string)`